db/db_impl.h

```diff
@@ -550,9 +550,18 @@ class DBImpl : public DB {
     WriteBatch* batch_;
     // The seq number of the first key in the batch
     SequenceNumber seq_;
+    // Number of sub-batched. A new sub-batch is created if we txn attempts to
+    // inserts a duplicate key,seq to memtable. This is currently used in
+    // WritePrparedTxn
+    size_t batch_cnt_;
```

Comment:
where is this being used?

Reply:
utilities/transactions/pessimistic_transaction_db.cc:147 in an assert statement.